Keas.Mvc/Controllers/ConfirmController.cs

```diff
@@ -45,6 +45,32 @@ public async Task<IActionResult> MyStuff()
             return View(viewmodel);
         }
 
+        public async Task<IActionResult> Landing()
+        {
+            var team = await _securityService.GetPerson(Team);
```

Comment:
@wongband do a check to see if `Team` is null/empty since there is no need to query the security service if `Team` is null.

Reply:
Yea I can do that

Reply:
If `Team` is null, where should I redirect the user?

Reply:
`_securityService.GetPerson(Team);` will always return a null team if the passed in `Team` is null, so there is no reason to call it, just fall through to show the user a list of their teams,